JASP-Engine/JASP/R/correlationbayesianpairs.R

```diff
@@ -762,7 +768,7 @@ CorrelationBayesianPairs <- function(dataset=NULL, options, perform="run", callb
 							# 
 							# plot[["data"]] <- .endSaveImage(image)
 							.plotFunc <- function() {
-								.plotPosterior.correlation(r=rs[i], n=ns[i], oneSided=oneSided, BF=BF10post[i], BFH1H0=BFH1H0, addInformation=options$plotPriorAndPosteriorAdditionalInfo, kappa=options$priorWidth,corCoefficient=options$corcoefficient)
+								.plotPosterior.correlation(r=rs[i], n=ns[i], oneSided=oneSided, BF=plotBF10post, BFH1H0=BFH1H0, addInformation=options$plotPriorAndPosteriorAdditionalInfo, kappa=options$priorWidth,corCoefficient=options$corcoefficient)
```

Comment:
What's the difference between BF10post[i] and plotBF10post?

Reply:
It is the exponent of logBF or otherwise just BF